bin/p2-rctl/main.go

```diff
@@ -153,6 +153,15 @@ func main() {
 	}
 }
 
+// SessionName returns a node identifier for use when creating Consul sessions.
+func SessionName() string {
+	hostname, err := os.Hostname()
+	if err != nil {
+		hostname = "unknown hostname"
+	}
+	return fmt.Sprintf("p2-rctl:%s:%s", hostname, time.Now().Format("0102-150405"))
```

Comment:
This date formatting is hard to sort lexicographically (ie. YYYY-mm-dd would be more amenable to sorting). Is that use case relevant here?